packages/mantine/src/templates/icons.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
```

Comment:
Rather than importing from `React`, only import what you need
```suggestion
import { ComponentPropsWithoutRef } from 'react';
```